{{cookiecutter.project_slug}}/.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,148 @@
+      NAME="Ubuntu"
+      VERSION="20.04.2 LTS (Focal Fossa)"
+      ID=ubuntu
+      ID_LIKE=debian
+      PRETTY_NAME="Ubuntu 20.04.2 LTS"
+      VERSION_ID="20.04"
+      HOME_URL="https://www.ubuntu.com/"
+      SUPPORT_URL="https://help.ubuntu.com/"
+      BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
+      PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
+      VERSION_CODENAME=focal
+      UBUNTU_CODENAME=focal
+
+      # macOS
+      $ system_profiler -detailLevel mini SPSoftwareDataType | head -n 6
+      Software:
+
+          System Software Overview:
+
+            System Version: macOS 10.15.7 (19H1323)
+            Kernel Version: Darwin 19.6.0
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Steps to Reproduce
+    description: >-
+      Describe exactly how to reproduce the problem, using a minimal test-case.
+      It would *really* help us understand your problem if you paste in the Python code
+      that you're running.
+
+
+      **HINT:** You can paste [GitHub Gist](https://gist.github.com) links for larger files.
+    value: |
+      <!--- Paste your minimal failing Python example code between the quotes below -->
+      ```python (paste below)
+
+      ```
+
+      <!--- ...or if you have a failing CLI command paste it between the quotes below -->
+      ```console (paste below)
+
+      ```
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: File Upload (optional)
+    description: >-
+      If your steps to reproduce your minimal failing example require either a spec or a
+      workspace file, please upload it by attaching it to the text area here.
+
+
+      **HINT:** You can paste [GitHub Gist](https://gist.github.com) links for larger files.
+    placeholder: >-
+      Attach any files or compressed archives by dragging & dropping, selecting,
+      or pasting them here.
+  validations:
+    required: false
+
+- type: textarea
+  attributes:
+    label: Expected Results
+    description: >-
+      Describe what you expected to happen when running the steps above.
+    placeholder: >-
+      I expected X to happen because I assumed Y.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Actual Results
+    description: >-
+      Paste verbatim program or command output.
+      Don't wrap it with triple backticks &mdash; your whole input will be
+      turned into a code snippet automatically.
+    render: console
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: {{ cookiecutter.project_name }} Version
+    description: >-
+      Paste verbatim output from `{{ cookiecutter.project_name }} --version` below, under the prompt line.
+      Don't wrap it with triple backticks &mdash; your whole input will be
+      turned into a code snippet automatically.
+    render: console
+    placeholder: |
+      $ {{ cookiecutter.project_name }} --version
```

Comment:
This might not work for every project. A better way to write this would be `python -m pip show {{ cookiecutter.project_name }}`